modules/ROOT/pages/amazon/amazon-sns-connector-design-center.adoc

```diff
@@ -0,0 +1,45 @@
+= Amazon SNS Design Center Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+The Flow Designer feature of Design Center enables you to create apps visually. To use Flow Designer, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the
+xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Flow Designer:
+
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
+
+For information on Flow Designer field values, see
+the xref:amazon/amazon-sns-connector-reference.adoc[Amazon SNS Connector Reference].
+
+== Configure the Input Source Trigger
+
+. In Design Center, click *Create*.
+. Click *Create new application* to open Flow Designer.
+. Specify a value for *Project name*, and click *Create*.
+. Click *Go straight to canvas* to exit from *Let's get started*.
+. Click on the trigger card.
+. Search for and select *HTTP Listener* or *Scheduler*.
+. In the *General* tab, enter the path.
+. Click off the trigger card.
+
+Flow Designer automatically saves all changes.
+
+== Configure the Target Component
+
+. Click *+* next to the trigger card.
+. In *Select a component*, search for the connector name.
+. Select the connector name.
+. Select an operation for the connector.
+. If needed, specify values that are required in the *General* tab.
```

Comment:
This is too general. We need to list what fields are required along with a description. The FD user is not going to enjoy jumping into the connector reference to figure out what should go in each field.

Reply:
I give an example of specific fields for an operation in the next steps. I will rewrite this it to make it more clear.